docs/reference/content/getting-started/async-quick-tour.md

```diff
@@ -0,0 +1,455 @@
+
+The MongoDB Async driver provides an asynchronous API that can leverage either Netty or Java 7's `AsynchronousSocketChannel` for fast and non-blocking IO.
+
+The API mirrors the new Synchronous API from the MongoDB Driver, but any methods that cause network IO take a `SingleResponseCallback<T>` and return immediately, where `T` is the type of response for the document.
+
+The `SingleResponseCallback<T>` interface requires the implementation of a single method `onResult(T result, Throwable t)` which is called when the operation has completed.  The `result` parameter contains the result of the operation, if successful. If the operation failed for any reason then the `t` contains the Throwable reason for the failure.
+
+{{% note class="important" %}}
+It's important to always check for errors in any `SingleResponseCallback<T>` implementation
+and handle them appropriately! Below the error checks are left out only for the sake of brevity.
+{{% /note %}}
+
+## Make a Connection
+
+The following example shows two ways to connect to the
+database `mydb` on the local machine.
+
+```java
+// Provide custom MongoClientSettings
+MongoClientSettings settings = MongoClientSettings.builder().build();
+MongoClient mongoClient = MongoClients.create(settings);
+
+// or use a connection string
+MongoClient mongoClient = new MongoClient(new ConnectionString("mongodb://localhost"));
+
+MongoDatabase database = mongoClient.getDatabase("mydb");
+```
+
+At this point, the `database` object will be a connection to a MongoDB
+server for the specified database.
+
+{{% note %}}
+There is no callback required for `getDatabase("mydb")` as there is no network IO required.
+A `MongoDatabase` instance provides methods to interact with a database
+but the database might not actually exist and will only be created on the
+insertion of data via some means; eg the creation of a collection or inserting documents,
+which do require callbacks as they require network IO.
+{{% /note %}}
+
+### MongoClient
+
+The `MongoClient` instance actually represents a pool of connections
+to the database; you will only need one instance of class
+`MongoClient` even with multiple threads.
+
+The `MongoClient` class is designed to be thread safe and shared among
+threads. Typically you create only 1 instance for a given database
+cluster and use it across your application.
+
+{{% note class="important" %}}
+When creating many `MongoClient` instances:
+
+-   All resource usage limits (max connections, etc) apply per
+    `MongoClient` instance
+-   To dispose of an instance, make sure you call `MongoClient.close()`
+    to clean up resources
+{{% /note %}}
+
+## Get a Collection
+
+To get a collection to operate upon, specify the name of the collection to
+the [`getCollection(String collectionName)`]({{< apiref "com/mongodb/client/MongoDatabase.html#getCollection-java.lang.String-">}})
+method:
+
+The following example gets the collection `test`. If the collection does not
```

Comment:
I know what you mean, but this is misleading.  It implies that calling the getCollection method itself will cause MongoDB to create it.